guides/common/modules/con_puma-threads.adoc

```diff
@@ -1,21 +1,13 @@
 [id="Puma_Threads_{context}"]
 = Puma threads
 
-Number of Puma threads (per Puma worker) is configured by using two values: `threads_min` and `threads_max`.
+To configure the number of Puma threads (per Puma worker), use these values: `threads_min` and `threads_max`.
 
 Value of `threads_min` determines how many threads each worker spawns at a worker start.
 Then, as concurrent requests are coming and more threads is needed, worker will be spawning more and more workers up to `threads_max` limit.
 
 We recommend setting `threads_min` to same value as `threads_max` as having fewer Puma threads lead to higher memory usage on your {ProjectServer}.
```

Comment:
Since https://projects.theforeman.org/issues/33214 (Foreman 3.1) this is the default behavior if you don't specify the min value.

Can we drop this line?